definitions/infra-unix_host/definition.yml

```diff
@@ -0,0 +1,11 @@
+domain: INFRA
```

Comment:
I'm unsure about using INFRA here (specially since there is an INFRA-HOST that arguably has overlap with this one even though we can't reuse that type because it generates custom identifiers). I would default this EXT here (this would mean changing this `domain` attribute, and the name of the folder containing these configs.

Reply:
I don't have a strong opinion either way on this, but I will point out that there's a precedent for using the `INFRA` domain on integrations data:
[Cassandra](https://github.com/newrelic-experimental/entity-synthesis-definitions/tree/main/definitions/infra-cassandranode)
[F5](https://github.com/newrelic-experimental/entity-synthesis-definitions/tree/main/definitions/infra-f5node)
[Kafka](https://github.com/newrelic-experimental/entity-synthesis-definitions/tree/main/definitions/infra-kafkabroker)
[MSSQL](https://github.com/newrelic-experimental/entity-synthesis-definitions/tree/main/definitions/infra-mssqlinstance)
[MySQL](https://github.com/newrelic-experimental/entity-synthesis-definitions/tree/main/definitions/infra-mysqlnode)
[NGINX](https://github.com/newrelic-experimental/entity-synthesis-definitions/tree/main/definitions/infra-nginxserver)
and more...

Reply:
Yeah, the reason we're a bit unsure about placing this in "INFRA" is because the existing domains have some historical implications tied to internal organization and governance, as well as some technicalities (e.g. many INFRA- entities generally get IDs that are generated internally in NR, and for synthesis we need to ensure that identifiers come directly from telemetry)  Let me get feedback from a couple of internal stakeholders so we can decide for a consistent rule to use.

Reply:
For context: There's been an internal decision to match guid formation to the agent reporting it, so we will have 1:1 instrumentation provider to entity type. (Prior we were considering having multiple agents report for a single entity type)
 
For this thread, my interpretation of that is we can push this to match the integration (eg `PROMETHEUS` or `PROM` for char limit) and let `infra` continue being an internal domain.

Reply:
Resolving this to clear the PR, I'll make a suggestion for EXT as we're using that pattern for most cases.